site/src/components/pages/hub/BlockDataContainer.tsx

```diff
@@ -43,7 +43,7 @@ export const BlockDataContainer: VoidFunctionComponent<
   const [blockState, setBlockState] = useState<Record<string, unknown>>({
     accountId: "test-account-id",
     entityId: "test-entity-id",
-    initialWidth: 300,
+    initialWidth: 600,
   });
```

Comment:
I've removed this in #143, `initialWidth` is not a common property to all blocks - can you update it in the example for the embed instead please?